lib/cli.js

```diff
@@ -24,20 +14,17 @@ var cmd = args[0];
 
 function pre() {
   if (opts.version) {
-    console.log(pkg.version);
-    return;
+    return console.log(pkg.version);
```

Comment:
return on a separate line makes the code more clear as it show the intent better